modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,6 +19,10 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-3644[OSSM-3644] Federation egress-gateway gets the wrong update of network gateway endpoints, causing extra endpoint entries. This has been updated on the server side so it gets the correct network gateway endpoints.
```

Comment:
- The bug fix description needs to be rewritten using the guidelines and template in the SSG: "Use past tense for the problem and present tense for the solution". See: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes
- I suggest rewording the second sentence to make specific what "this" and "it" refer to. It's not clear right now. "This" what? "This issue"? What precisely "gets the correct network gateway endpoints"?